manifests/params.pp

```diff
@@ -0,0 +1,22 @@
+# @summary Provide OS conditional parameters for class sssd
```

Comment:
params is outdated and has already been deprecated. please put these values in hiera under `data/`

Reply:
Good idea!

Reply:
Done.